app/app.go

```diff
@@ -250,6 +250,7 @@ func New(
 	appOpts servertypes.AppOptions,
 	baseAppOptions ...func(*baseapp.BaseApp),
 ) *App {
+	var err error
```

Comment:
Since we declare the variable here, shouldn't we replace "err :=" with "err =" everywhere?